tests/test_import_export.py

```diff
@@ -0,0 +1,332 @@
+"""
+Tests that data is retained after an import/export or export/import cycle.
+"""
+
+from __future__ import unicode_literals
+from __future__ import print_function
+
+from lxml import etree
+import os
+from shutil import rmtree
+from subprocess import check_call
+from tempfile import mkdtemp
+from unittest import TestCase
+
+from xbundle import XBundle
+from tests.util import clean_xml, file_from_string
+
+
+class TestImportExport(TestCase):
+    """
+    Test that data is retained after an import/export or export/import cycle.
+    """
+    def test_export_import(self):
+        """
+        Test export then import.
+        """
+        bundle = XBundle()
+        cxmls = """
+<course semester="2013_Spring" course="mitx.01">
```

Comment:
For all of these outdented XML docstrings:

Feel free to ignore this, but I'd rather see them declared in another file and used here, or declared at the bottom of the file as strings and just used here.

Rationale: The longer they are, the more they clutter up the code when reading, and reading the content of them is rarely useful for reading and understanding what the test is doing.

Also, this makes them easier to re-use, which may sound like YAGNI, but _if_ they _do_ get re-used, any future modifications made for one test could expose a flaw in another test.